### ISSUE TITLE
Lost all my favorites and history

### ISSUE THREAD

**Issue by avently** [segler-alex/RadioDroid#305]
I had a version dated 6 February from my last pull request. Today I merged the latest changes and all my favorites and history are gone. Settings is untouched. I tried to restore it from backup but situation is the same. What's happened and why?

**Comment by avently** [comment]
@segler-alex this commit https://github.com/segler-alex/RadioDroid/commit/e8c65e86a4829859058597b705d1c6d5fc29779e is the cause of the issue. I see this error in logcat when opening favorites or history tabs: 

```
net.programmierecke.radiodroid2 E/DATAStation: DecodeJson() #2 org.json.JSONException: No value for clicktrend
```
It means every user have lost history and favorites.

**Comment by Wikinaut** [comment]
I have the same when upgrading to 0.54

**Comment by avently** [comment]
@Wikinaut I already fixed it myself but the situation when the owner doesn't care about that is not what I expected. Maybe he is busy at work or something else

**Comment by Wikinaut** [comment]
@avently How exactly did you fix that (can you send a pull request)?

**Comment by avently** [comment]
@Wikinaut I'll do it later today

**Comment by avently** [comment]
@Wikinaut fixed here #319
